packages/medusa/src/repositories/gift-card.ts

```diff
@@ -40,7 +39,7 @@ export class GiftCardRepository extends Repository<GiftCard> {
     const entitiesAndRelations = entitiesIdsWithRelations.concat(entities)
 
     const entitiesAndRelationsById = groupBy(entitiesAndRelations, "id")
-    return map(entitiesAndRelationsById, entityAndRelations =>
+    return map(entitiesAndRelationsById, (entityAndRelations) =>
       merge({}, ...entityAndRelations)
```

Comment:
Suggestion: I think that we should use native method when possible to rely as less as possible on external package. I would suggest the spread operator here such as { ... } wdyt?